src/js/actions/superselect.js

```diff
@@ -660,7 +661,7 @@ define(function (require, exports) {
                 return descriptor.playObject(toolLib.setToolOptions("moveTool", { "$Abbx": false }))
                     .bind(this)
                     .then(function () {
-                        return this.transfer(click, doc, x, y, diveIn, false);
+                        return this.transfer(click, doc, x, y, diveIn, false, true);
```

Comment:
It seems suboptimal to transfer to an action (`click`) that starts out by performing a hit test since we already did the hit test. Maybe a subprocedure could be factored out of `click` which takes the hit layers (or more!?) as parameters, and then both `click` and `drag` could transfer to that subprocedure? Seems like we could save at least one redundant hit test that way.